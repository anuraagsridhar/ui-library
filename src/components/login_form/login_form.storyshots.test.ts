
import initStoryshots from '@storybook/addon-storyshots';

const options = {
  configPath: '.storybook',
  storyKindRegex: /^components\.login_form$/,
};

initStoryshots(options);
