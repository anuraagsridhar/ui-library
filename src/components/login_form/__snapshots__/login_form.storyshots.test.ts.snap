// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots components.login_form Static Examples 1`] = `
<section>
  <fieldset>
    <div>
      <div
        className=""
      >
        <div>
          <div>
            Login
          </div>
        </div>
        <div>
          <div>
            <div>
              Username
            </div>
            <div>
              <input
                onChange={[Function]}
                placeholder="Enter username"
                type="text"
                value="asridhar"
              />
            </div>
          </div>
          <div>
            <div>
              Password
            </div>
            <div>
              <input
                onChange={[Function]}
                placeholder="Enter password here:"
                type="password"
                value="testpass"
              />
            </div>
          </div>
          <div>
            <div
              className=""
              onClick={[Function]}
            >
              <div>
                Submit
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </fieldset>
</section>
`;
